Modix/Controllers/ChannelController.cs

```diff
@@ -34,14 +35,14 @@ public async Task<IActionResult> ChannelDesignations()
         }
 
         [HttpDelete("{id}")]
-        public async Task<IActionResult> RemoveDesignation(long id)
+        public async Task<IActionResult> RemoveDesignationAsync(long id)
         {
-            await ChannelService.RemoveDesignatedChannelByIdAsync(id);
+            await _channelService.RemoveDesignatedChannelByIdAsync(id);
             return Ok();
         }
 
         [HttpPut]
-        public async Task<IActionResult> CreateDesignation([FromBody] DesignatedChannelCreationData creationData)
```

Comment:
`[FromBody]` is not necessary?

Reply:
Not anymore with the addition of the `[ApiControler]` attribute. Handles binding, and Model State validation for you now. See - https://www.strathweb.com/2018/02/exploring-the-apicontrollerattribute-and-its-features-for-asp-net-core-mvc-2-1/